From 430fdbe3d3107b723ecc8d482d7bd665cc3afae3 Mon Sep 17 00:00:00 2001
From: sushant suryawanshi <sushant.suryawanshi@rajasoftwarelabs.com>
Date: Tue, 27 Jan 2026 11:43:24 +0530
Subject: [PATCH] added test

---
 hello.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hello.java b/hello.java
index 98d80e8..3d5b3ac 100644
--- a/hello.java
+++ b/hello.java
@@ -1,7 +1,7 @@
 
 public class hello {
     public static void main(String[] args) {
-        System.out.println("dev");
+        System.out.println("test");
     }
     
 }
-- 
2.50.1 (Apple Git-155)

